docs/data-sources/datasets.md

```diff
@@ -0,0 +1,29 @@
+# Data Source: honeycombio_datasets
+
+Search datasets.
+
+## Example Usage
+
+```hcl
+# returns all datasets
+data "honeycombio_datasets" "my_datasets" {
+}
+
+# only return the datasets starting with 'foo_'
+data "honeycombio_datasets" "foo_datasets" {
+  starts_with = "foo_"
+}
+```
```

Comment:
Convenient.

If anyone from HC is taking API feedback here, it'd be nice if there was some kind of tagging functionality for datasets.

For instance if I want to add an alert to all the `prod` `web` datasets (which you could probably do via convention with prefixes)

Reply:
Agree this would be useful! I've copied your feedback to the Pollinators Slack 🙂